android/src/com/mapswithme/maps/bookmarks/BookmarksCatalogFragment.java

```diff
@@ -268,7 +271,28 @@ public boolean shouldOverrideUrlLoading(WebView view, String url)
       if (fragment == null)
         return false;
 
-      return fragment.downloadBookmark(url);
+      boolean result = fragment.downloadBookmark(url);
+
+      Uri uri = Uri.parse(url);
+      List<String> pathSegments = uri.getPathSegments();
+      for (String each : pathSegments)
+      {
+        if (TextUtils.equals(each, SUBSCRIBE_PATH_SEGMENT))
+        {
+          openSubscriptionScreen();
+          return result;
```

Comment:
Тут return true должно быть, т.к. мы сами обрабатываем этот урл иначе вебью тоже по нему перейдет и там будет скорее всего 404, этот урл исключительно для нативной части приложения, на сервере по нему ничего не откроется.

True if the host application wants to leave the current WebView and handle the url itself, otherwise return false.